src/Cassandra/Statement.cs

```diff
@@ -48,7 +48,7 @@ public virtual object[] QueryValues
 
         /// <summary>
         /// Gets the serial consistency level for this query.
-        /// </summary>        
+        /// </summary>
```

Comment:
where this changes come from ?
I agree we need to clean up, but let's do it with some tool and within a separate PR